migrations/v7-MIGRATION.md

```diff
@@ -0,0 +1,44 @@
+# V7 API Migration Guide
+
+This latest release updates the SDK to use BillingClient 6. This version of BillingClient brings little change compared 
+with BillingClient 5 which brought an entire new subscription model which resulted in large changes across the entire SDK.
+
+The only modification at the API level involves replacing "ProrationMode" with "ReplacementMode". The specific replacement 
+modes remain unchanged. 
+
+However, there is a behavior change in the "DEFERRED" replacement mode. Previously, a purchase would only occur after the
+product change. In BC6, the new purchase occurs place immediately, while the product change becomes effective when the old
+item expires. Until we can properly support this new behaviour, we have disabled the "DEFERRED" replacement mode and it 
+will be added back in a future release of the SDK. 
```

Comment:
Let's make it the implications for a developer super explicit here: I might not know whether this affects me at all, and it most likely doesn't. 

If your app supports product changes using DEFERRED replacement mode (link to docs), then you can either stick with the previous major version until support for DEFERRED is re-introduced in this major version, or you can remove DEFERRED replacement options from your app. 

If your app doesn't currently use DEFERRED replacement modes, then you should be safe to upgrade to this version without changes in behavior.

Reply:
or maybe flipping the order of those since DEFERRED isn't very widely used